R/check-superseded-deps.R

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
not even sure "superseded" is the right name! I wanted something not too negative.